biocrnpyler/species.py

```diff
@@ -702,58 +771,64 @@ class OrderedPolymerSpecies(OrderedComplexSpecies,OrderedPolymer):
     sometimes it is convenient to pass an internal Species. Both will work from the point of view
     of any Mechanism.
     """
-    default_material="ordered_polymer"
-    def __init__(self,species, name=None, base_species = None, material_type = default_material, \
-                             attributes = None,circular = False):
+    default_material = "ordered_polymer"
+
+    def __init__(self, species, name=None, base_species=None, material_type=default_material,
+                 compartment=None, attributes=None, initial_concentration=0, circular=False):
```

Comment:
initial_concentration is coming back. Please double check this is removed everywhere.